packages/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -2,23 +2,27 @@ import * as ProgressPrimitive from "@radix-ui/react-progress";
 import { type ComponentPropsWithRef, type ElementRef, forwardRef } from "react";
 
 import { Box } from "../box";
+import { type ExtendProps } from "../utils";
 
-type ProgressBarProps = ComponentPropsWithRef<typeof ProgressPrimitive.Root>;
+type ProgressBarProps = ExtendProps<
+  ComponentPropsWithRef<typeof ProgressPrimitive.Root>,
+  Pick<ComponentPropsWithRef<typeof Box>, "w">
+>;
 
 export const ProgressBar = forwardRef<
   ElementRef<typeof ProgressPrimitive.Root>,
   ProgressBarProps
->((props, ref) => {
+>(({ w = "2/3", ...props }, ref) => {
   const widthPercentage = ((props.value ?? 0) / (props.max ?? 100)) * 100;
 
   return (
-    <Box asChild border="1" h="6" overflow="hidden" w="1/2">
+    <Box asChild border="1" h="6" overflow="hidden" w={w}>
```

Comment:
This shouldn’t be just `w` but we should forward/spread all props here

Reply:
Bhai, just to clarify, by forwarding all props, you mean I should forward all the props that `Box` accepts to the `ProgressBar.Root`?

Reply:
yep
```suggestion
    <Box asChild border="1" h="6" {...props}>
```

Reply:
Addressed in [3b33ac6](https://github.com/optimizely-axiom/optiaxiom/pull/71/commits/3b33ac6a39f747ac072cebdeacdafb2cff6c7c93). Kept `h=6`, `overflow=hidden` (otherwise it will overflow if `value` > `max`. Alternatively, I can handle it in JS) as default, but user will be able to override those. Does that look okay bhaia?

Reply:
why would `value` be bigger than `max`?

radix throws a prop warning in such cases:

<img width="1264" alt="Screenshot 2024-06-06 at 12 54 31 PM" src="https://github.com/optimizely-axiom/optiaxiom/assets/758244/573aab36-dda4-48ae-b87a-6eefe59ef8ac">

sorry i didn't notice this in the initial PR - but we should not show an indicator at all if the `value` exceeds `max`

it's fine to leave in `overflow: hidden` but we should not rely on it to cap the indicator - since it leads to weird issues with the border-radius:

### proper 100% state
<img width="122" alt="Screenshot 2024-06-06 at 12 58 46 PM" src="https://github.com/optimizely-axiom/optiaxiom/assets/758244/eb412edb-b6d2-4955-b22f-0259a32ced85">

### overflow hidden with value > max
<img width="122" alt="Screenshot 2024-06-06 at 12 58 40 PM" src="https://github.com/optimizely-axiom/optiaxiom/assets/758244/195623f4-06b5-4dc9-b104-1819f82d2300">